scripts/ansible/roles/linux/openenclave/vars/ubuntu/main.yml

```diff
@@ -3,15 +3,14 @@
 
 ---
 cmake_target_version: "3.13.1"
-clang_target_version: "8.0.0"
+clang_target_version: "10.0.1"
```

Comment:
Can you double check if it's 10.0.0 or 10.0.1?

Reply:
On window we are using 10.0.0 and on Linux we are using 10.0.1. On windows 10.0.1 is not available. @mingweishih Let us know if we are okay with this inconsistency.

Reply:
I verified, and we should be using 10.0.1 in Ubuntu.